docs/UserGuide.md

```diff
@@ -539,6 +539,21 @@ Imports a list of contacts from a .csv file.
 * `import fp/data/data.csv` will import all valid entries from the `data.csv` folder in the `/data` directory of the NUS Classes folder.
 
 
+
+### User-friendly date display
```

Comment:
Should we put this at the top? Under Features or a separate section by itself

Reply:
i'll leave it to you, currently it's at the bottom of "other features" which seems appropriate to me

Reply:
Ok then, I think that does make sense